staging/src/k8s.io/code-generator/cmd/client-gen/generators/fake/generator_fake_for_type.go

```diff
@@ -327,7 +330,7 @@ var listSubresourceTemplate = `
 func (c *Fake$.type|publicPlural$) List($.type|private$Name string, opts $.ListOptions|raw$) (result *$.resultType|raw$List, err error) {
 	obj, err := c.Fake.
 		$if .namespaced$Invokes($.NewListSubresourceAction|raw$($.type|allLowercasePlural$Resource, $.type|private$Name, "$.subresourcePath$", $.type|allLowercasePlural$Kind, c.ns, opts), &$.resultType|raw$List{})
-		$else$Invokes($.NewRootListAction|raw$($.type|allLowercasePlural$Resource, $.type|allLowercasePlural$Kind, opts), &$.resultType|raw$List{})$end$
+		$else$Invokes($.NewRootListSubresourceAction|raw$($.type|allLowercasePlural$Resource, $.type|private$Name), &$.resultType|raw${})$end$
```

Comment:
missed `$List{}` in the end. but agree with @liggitt, subresource doesn't support list